stl/inc/xstring

```diff
@@ -2361,8 +2369,18 @@ public:
     using const_reverse_iterator = _STD reverse_iterator<const_iterator>;
 
 private:
-    static constexpr auto _BUF_SIZE   = _Scary_val::_BUF_SIZE;
-    static constexpr auto _ALLOC_MASK = _Scary_val::_ALLOC_MASK;
+    static constexpr size_type _BUF_SIZE              = _Scary_val::_BUF_SIZE;
+    static constexpr size_type _ALLOC_MASK            = _Scary_val::_ALLOC_MASK;
+    static constexpr size_type _SMALL_STRING_CAPACITY = _Scary_val::_SMALL_STRING_CAPACITY;
+
+    _NODISCARD static _CONSTEXPR20 bool _Entails_large_string(size_type _Size_or_capacity) noexcept {
+        // decides whether planned size/capacity entails large mode
+        return _Size_or_capacity > _SMALL_STRING_CAPACITY;
+    }

```

Comment:
Are there benefits to make trivial internal methods `_CONSTEXPR20` instead of `constexpr`? I see `_Large_string_engaged` use `_CONSTEXPR20`.